backend/clap/proto/data_store.proto

```diff
@@ -182,6 +182,20 @@ message StudyVisitRequest {
     int32 submission_id = 2;
 }
 
+message EmailDetailsRequest {
+    int32 assessor_id         = 1;
```

Comment:
I don't get this... where is clap supposed to take `assesor_id` or `project_id` from ?